bentoml/_internal/io_descriptors/pandas.py

```diff
@@ -176,18 +240,22 @@ def __init__(
         self._enforce_dtype = enforce_dtype
         self._shape = shape
         self._enforce_shape = enforce_shape
-        self._mime_type = "application/json"
+        self._default_format = SerializationFormat[default_format.upper()]
 
     def openapi_schema_type(self) -> t.Dict[str, t.Any]:
         return _schema_type(self._dtype)
 
     def openapi_request_schema(self) -> t.Dict[str, t.Any]:
         """Returns OpenAPI schema for incoming requests"""
-        return {self._mime_type: {"schema": self.openapi_schema_type()}}
+        return {
+            f'request.headers["content-type"] (default: {self._default_format.mime_type})': {
```

Comment:
I was a little unsure how to handle the dynamic `content-type` here. I figured that since the key of the outer dict is `str`, I could put anything in there, so I chose something descriptive.

Reply:
I don't think this follows OpenAPI convention. Refers to [here](https://swagger.io/docs/specification/media-types/)